go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/sirupsen/logrus v1.0.5
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.1
+	github.com/stretchr/testify v1.3.0
```

Comment:
Can we implement `Contains` in our check repo?

Reply:
+1, I prefer not to introduce too many testing frameworks...